### PR TITLE
Fix infinite skeleton on first release

### PR DESCRIPTION
**Pull request by nachoiacovino**

### CODE REVIEW

File: components/contract-components/contract-table/cells/released-by.tsx [hunk 1]
```diff
@@ -26,8 +26,17 @@ export const ContractReleasedByCell: React.FC<
       .publisher;
 
   const releaserEnsQuery = ens.useQuery(releaser);
+
+  console.log({ isSuccess: fullPublishMetadata, isPrebuilt });
```
Comment (jnsdls) [inline components/contract-components/contract-table/cells/released-by.tsx:30]: let's remove this

File: components/contract-components/contract-table/cells/released-by.tsx [hunk 2]
```diff
@@ -26,8 +26,17 @@ export const ContractReleasedByCell: React.FC<
...
+
   return (
-    <Skeleton isLoaded={fullPublishMetadata.isSuccess || isPrebuilt}>
+    <Skeleton
+      isLoaded={
+        fullPublishMetadata.isSuccess ||
+        fullPublishMetadata.isStale ||
```
Comment (jnsdls) [inline components/contract-components/contract-table/cells/released-by.tsx:36]: why does the `isStale` get hit?

File: components/contract-components/contract-table/cells/description.tsx [hunk 3]
```diff
@@ -23,12 +23,18 @@ export const ContractDescriptionCell: React.FC<
     BuiltinContractMap[value as keyof typeof BuiltinContractMap]?.description;
 
   return (
-    <Skeleton isLoaded={fullPublishMetadata.isSuccess || !!description}>
+    <Skeleton
+      isLoaded={
+        fullPublishMetadata.isSuccess ||
+        fullPublishMetadata.isStale ||
```
Comment (jnsdls) [inline components/contract-components/contract-table/cells/description.tsx:29]: why does the isStale get hit, how does this fix it?

**Review by jnsdls — CHANGES_REQUESTED** [review]
remove the log and explain why `isStale` fixes it pls

File: components/contract-components/contract-table/cells/released-by.tsx (hunk 2 above)
Comment (nachoiacovino) [reply components/contract-components/contract-table/cells/released-by.tsx:36]: so we never get fullPublishMetadata if the contract has never been released before, is there a better way to check this?

Comment (jnsdls) [reply components/contract-components/contract-table/cells/released-by.tsx:34]: ah, as in we never try to query it or we try to query but it returns a 404?

stale => there is data in the cache but it needs to be re-fetched (which seems like the wrong thing to check)

we could check isIdle *maybe*? or if we try to query and it's a 404 we can check the error?

Comment (nachoiacovino) [reply components/contract-components/contract-table/cells/released-by.tsx]: I just updated to check that we're not fetching, this works better I can try again with isIdle see if that works better, I'll brb in 30 mins

**Review by jnsdls — APPROVED** [review]
this should work now, @nachoiacovino take a look if isIdle might be better still but will get this in for now

**Pull request merged**